index.bs

```diff
@@ -1202,17 +1203,18 @@ runs these steps:
        {{DOMException}} of |accessResult|'s
        [=file system access result/error name=] and abort these steps.
 
-    1. Let |command| be |input|.{{WriteParams/type}} if |input| is a {{WriteParams}},
-       and {{WriteCommandType/"write"}} otherwise.
+    1. Let |command| be |input|'s {{WriteParams/type}} if
+       |input| is a {{WriteParams}}; otherwise {{WriteCommandType/"write"}}.
```

Comment:
is a [=/dictionary=]*

We don't know the dictionary type at this point. Existing issue though so feel free to file a follow-up. (Or I could.)

Reply:
Sounds like we can't check the specific type (i.e. WriteParams) of a union? Just the data structure (i.e. dictionary/boolean/etc)

Would you mind filing a follow-up?